src/components/Livestream.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components'
+import { Card } from '../components/Common.js'
+import { H2 } from '../components/Typography'
+
+const TwitchFrame = styled.iframe`
+  border-radius: 5px;
+  border: none;
+`
+
+const LivestreamContainer = styled(Card)`
```

Comment:
Seems like we have a lot of names for a container, sometimes its `Container`, or `Div`, or `Card`

Reply:
what do you suggest we standardize it to

Reply:
I think if its actually using `CardLike` or the actual card component use Card, everything else use Container?

Reply:
Mind renaming `TimeDiv` and anything else? :*

Reply:
will do